platformio.ini

```diff
@@ -195,325 +173,295 @@ build_flags =
   -DMIMETYPE_MINIMAL
   ; other special-purpose framework flags (see https://docs.platformio.org/en/latest/platforms/espressif8266.html)
   ; decrease code cache size and increase IRAM to fit all pixel functions
-  -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
-  ; -D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
+  -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48 ;; in case of linker errors like "section `.text1' will not fit in region `iram1_0_seg'"
+  ; -DPIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM48_SECHEAP_SHARED ;; (experimental) adds some extra heap, but may cause slowdown
 
 lib_deps =
-  #https://github.com/lorol/LITTLEFS.git
+  ${env.lib_deps}
   ESPAsyncTCP @ 1.2.2
   ESPAsyncUDP
-  ${env.lib_deps}
```

Comment:
This order - specific libs first, `${env.lib_deps}` last - was chosen on purpose.
The reason is that asyncWebserver imports an old ESPAsyncTCP release. so the only way to ensure that only the version we want is used, was to put "our" ESPAsyncTCP before asyncWebserver which is part of `${env.lib_deps}`.

Pease keep the previous order.